handbook/company/product-groups.md

```diff
@@ -16,44 +16,48 @@ At Fleet, [anyone can contribute](https://fleetdm.com/handbook/company#openness)
 
 | Product group             | Goal _(value for customers and/or community)_                       | Capacity\* |
 |:--------------------------|:--------------------------------------------------------------------|:-----------------|
-| [MDM](#mdm-group)                                       | Reach maturity in the "MDM" product category. | 65       |
-| [Customer experience (CX)](#customer-experience-group)  | Make customers happier and more successful.   | 78       |
+| [Endpoint ops](#endpoint-ops-group)                     | Make customers happier and more successful.             | 78       |
```

Comment:
```suggestion
| [Endpoint ops](#endpoint-ops-group)                     | Increase and exceed maturity in the "Endpoint operations" category.             | 78       |
```

We're updating the goal, right? https://github.com/fleetdm/fleet/pull/14589/files#diff-c99d12c3af50c0c2aca2b9ef7597c02ccfe87678291956ff0b2e83d63978ea38R28

Reply:
Mike: Yes